application/models/FailedLoginAttempt.php

```diff
@@ -91,17 +91,20 @@ public function isLockedOut(string $attemptType): bool
 
         switch ($attemptType) {
             case FailedLoginAttempt::TYPE_LOGIN:
-                $timeOut = Yii::app()->getConfig('timeOutTime');
-                $maxLoginAttempt = Yii::app()->getConfig('maxLoginAttempt');
+                $timeOut = floatval(App()->getConfig('timeOutTime'));
```

Comment:
Float? Why not int?

Reply:
Why int ? why not 500ms ;) (but right : not really work since we use time)

Reply:
PS : i can do it work : 0.5 or 2.650 if needed. Not complex.

Reply:
The amount already has a measuring unit, like seconds or minutes. Usually that is expressed with an int.
I don't see a reason for adding more complexity to this.

Reply:
OK, i update